changelog/extern-std-cpp20.dd

```diff
@@ -0,0 +1,5 @@
+Added compiler support for `-extern-std=c++20`
```

Comment:
Maybe make the title a bit more accurate, as people might have expectations w.r.t. to coroutines and whatnot.

E.g.
```suggestion
The compiler now accepts `-extern-std=c++20`
```

Reply:
It's pretty clear that `-extern-std=` is for mangling, so the only thing on the table AFAIAC is `char8_t`, and how we are going to support mangling for that type, granted there's already the following mappings:
- `char` -> `char`
- `wchar` -> `char16_t`
- `dchar` -> `char32_t`
- `byte` -> `signed char`
- `ubyte` -> `unsigned char`

Reply:
(Done, by the way).